tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6621,13 +6613,45 @@ Status ConvertResize(OpConverterParams* params) {
   // Calculate output dimensions.
   // Given input dimensions [N, C, H, W] and output size [H_out, W_out],
   // output dimensions equals [N, C, H_out, W_out]
-  nvinfer1::Dims output_dimensions;
-  output_dimensions.nbDims = inputs_tensor->getDimensions().nbDims;
-  for (int i = 0; i < output_dimensions.nbDims; ++i) {
-    output_dimensions.d[i] = inputs_tensor->getDimensions().d[i];
+  const auto inputs_dims = inputs_tensor->getDimensions();
+  nvinfer1::Dims output_shape_dims;
+  ITensorProxyPtr output_shape_tensor;
```

Comment:
This line should go inside the else branch below.

Reply:
Both output_shape_dims and output_shape_tensor need to be on this outer scope because these objects are used on the other if statement below.

Reply:
Same, the comment assume we merge the two if-stmts.